pkg/watch/mux.go

```diff
@@ -39,6 +39,8 @@ const incomingQueueLength = 25
 // Broadcaster distributes event notifications among any number of watchers. Every event
 // is delivered to every watcher.
 type Broadcaster struct {
+	// TODO: see if this lock is needed now that new watchers go through
+	// the incoming channel.
```

Comment:
If `stress` runs these tests 10,000 times without flakes, would you say this TODO is safe to remove?